### ISSUE TITLE
Blocky color shifts in the editor background on displays with custom color profiles

### ISSUE THREAD

**Issue by hpohlmeyer** [atom/atom#14649]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

I have color shifting color blocks in the background of the editor. This issue only appears, when I use a custom monitor profile. When I switch to the _Color-LCD_ profile, which is the default monitor profile for my MacBook, the blocks disappear. However, the issue appears for every custom color profile. 

This issue is present in the current beta, as well as atom stable since a long time.
The color blocks change in shape and size, depending on what is shown in the editor.
It’s especially noticable on dark backgrounds.

But images say more than a thousand words:

__Atom Beta with packages__
![](https://cloud.githubusercontent.com/assets/6564077/26487302/3cc036fc-41ff-11e7-9bcf-c4b11e20ac12.png)
![](https://cloud.githubusercontent.com/assets/6564077/26487303/3cc2dc68-41ff-11e7-98eb-01f6aeced1ed.png)

__Atom Beta in safe-mode__
![safe mode](https://cloud.githubusercontent.com/assets/6564077/26487301/3ca68d38-41ff-11e7-8a99-3e671c1296c1.png)

### Steps to Reproduce

1. Open Atom

**Expected behavior:** The background is rendered in a single color

**Actual behavior:** The background includes blocky color-shifts

**Reproduces how often:** 100%

### Versions

```
Atom    : 1.18.0-beta1
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
MacOS   : 10.12.4
```

### Additional Information

I noticed this issue some time earlier in an earlier slack beta, so this might be an electron issue.

__The color profiles I use__
[ColorProfiles.zip](https://github.com/atom/atom/files/1031305/ColorProfiles.zip)

The issue is not exclusive to the custom color profiles though. If you switch to _Adobe RGB (1998)_ the same issue appears.

**Comment by dackmin** [comment]
probably a deeper understanding of #14646

**Comment by fiso** [comment]
@dackmin Yeah I can confirm that my issue is, like this one states, dependent on color profiles. I'll close mine as a duplicate.

**Comment by thomasjo** [comment]
Duplicate of https://github.com/atom/atom/issues/12634. Will be fixed once #12696 ships, which will most likely be Atom 1.19.0.
